test/integration/lib/config/foxbox.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+
+	nupnp_server : {
+		id :"111788fffe230b96",
+		port: "8002"
+	},
+
+	credential :  {
+		"email": "npark@mozilla.com",
+		"username": "admin",
+		"password": "00000000"
+	},
+
+	foxbox : { 
+		url : "http://localhost:3000"
+	},
+
+	philips_hue : {
+	port : 8001
+	},
+
+	header : {
+      "Content-Type" : "application/json"
```

Comment:
Nit: I don't remember, but I probably changed my mind since the last review. I don't see a use case where the header is depdendant of the environment. I'd move it to a js file instead.